src/casp/bq_scripts/create_bq_tables.py

```diff
@@ -102,7 +98,27 @@ def create_bigquery_objects(client, project, dataset):
             bigquery.SchemaField("cas_cell_index", "INTEGER", mode="REQUIRED"),
             bigquery.SchemaField("original_cell_id", "STRING", mode="REQUIRED"),
             bigquery.SchemaField("cell_type", "STRING", mode="REQUIRED"),
-            bigquery.SchemaField("obs_metadata", "JSON", mode="REQUIRED"),
+            # Data parsed from cellxgene
```

Comment:
This schema also needs to be refactored out to a more accessible place for better code readability.